common/remote_object.go

```diff
@@ -15,6 +16,8 @@ import (
 	"github.com/dop251/goja"
 )
 
+var bigIntR = regexp.MustCompile("^[0-9]*n$")
```

Comment:
Should we make the name more explicit, maybe like `bigIntRegEx`?

Reply:
Resolved in 99009e552ca56d8fd7050f3dc807eba4afe14369